app/(dashboard)/page.test.ts

```diff
@@ -1,5 +1,8 @@
 import { shortest } from "@antiwork/shortest";
 
+const loginEmail = `shortest@${process.env.MAILOSAUR_SERVER_ID}.mailosaur.net`;
+if (!loginEmail) throw new Error("MAILOSAUR_LOGIN_EMAIL is required");
+
 shortest("Verify that buttons on the landing page are rounded");
-shortest("Login using this email: mission-health@rdt7stzf.mailosaur.net");
+shortest("Log in", { email: loginEmail });
 shortest("Verify that the user can access the /dashboard page");
```

Comment:
This test doesn't seem to work as the user with Mailosaur email doesn't have GitHub connected, and the AI is confused by the Dashboard page:

![CleanShot 2025-01-03 at 20 11 52@2x](https://github.com/user-attachments/assets/fbb5abdc-e417-4d14-bb5c-3b762b5142fb)

Unsure what would be a good way to fix it. Maybe change the test, or just delete for now?

Reply:
We can delete the test for now.